persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/EventSourcedActorOf.scala

```diff
@@ -158,30 +156,46 @@ object EventSourcedActorOf {
 
   implicit final private class EventStoreOps[F[_], E](val store: EventStore[F, E]) extends AnyVal {
```

Comment:
MiMa barks on changes in `private` class thus disabled for now

Reply:
bumping major version works...

Reply:
I found the above workaround in https://docs.scala-lang.org/overviews/core/binary-compatibility-for-library-authors.html

Reply:
It works! Many thanks for the workaround 🙇